offchain/indexer/Cargo.toml

```
@@ -13,6 +13,7 @@ test = false
 http-health-check = { path = "../http-health-check" }
 rollups-data = { path = "../data" }
 rollups-events = { path = "../rollups-events" }
+log = {path = "../log"}
```

Also keeping them in alphabetic order:
```suggestion
log = { path = "../log" }
rollups-data = { path = "../data" }
rollups-events = { path = "../rollups-events" }
```

---

Done at 9e1b6f8e7bc477c334f4e109cf3bb7a1642bc44d